board/safety/safety_volkswagen.h

```diff
@@ -93,16 +105,25 @@ static int volkswagen_tx_hook(CAN_FIFOMailBox_TypeDef *to_send) {
 
     // reset to 0 if either controls is not allowed or there's a violation
     if (violation || !controls_allowed) {
-      vw_desired_torque_last = 0;
-      vw_rt_torque_last = 0;
-      vw_ts_last = ts;
+      volkswagen_desired_torque_last = 0;
+      volkswagen_rt_torque_last = 0;
+      volkswagen_ts_last = ts;
     }
 
     if (violation) {
       tx = 0;
     }
   }
 
+  // FORCE CANCEL: ensuring that only the cancel button press is sent when controls are off.
+  // This avoids unintended engagements while still allowing resume spam
+  if ((addr == MSG_GRA_ACC_01) && !controls_allowed && (bus == 2)) {
```

Comment:
Proposed for clarity:
```  if ((bus == 2) && (addr == MSG_GRA_ACC_01) && !controls_allowed) {```
And similar for bus 0/HCA_01 earlier in the function.

Reply:
ok

Reply:
I'll keep the bus check on HCA_01 since it's not necessary, as we discussed